antismash/outputs/html/css/secmet.scss

```diff
@@ -0,0 +1,276 @@
+/* colour definitions for secondary metabolite types */
+
+@mixin secmet($fg: black, $bg: white, $highlight: orangered){
+    background-color: $bg;
+    a, a:link, a:visited {
+        color: $fg;
+    }
+    rect& {
```

Comment:
I don't know why this line highlights red, but `rect&` generates correctly as `rect.NRPS` etc, where `rect.&` does not.